src/main/java/org/zalando/nakadi/repository/kafka/KafkaLocationManager.java

```diff
@@ -77,8 +80,10 @@ private static String buildBootstrapServers(final List<Broker> brokers) {
     private Properties buildKafkaProperties(final List<Broker> brokers) {
         final Properties props = new Properties();
         props.put("bootstrap.servers", buildBootstrapServers(brokers));
-        props.put("value.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
-        props.put("key.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
+        props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG,
+                "org.apache.kafka.common.serialization.ByteArrayDeserializer");
+        props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG,
+                "org.apache.kafka.common.serialization.ByteArrayDeserializer");
```

Comment:
you are right, they are overridden, I did not want to refactor that class at all, but @lmontrieux asked for changing properties to use kafka clients constants. If you insist I can factor this out.